### PR TITLE
Speaker id db implementation

### PR DESCRIPTION
**Pull request by staffanru**
This adds speaker id generation to the db calls

### CODE REVIEW

**Review by SmariF89 — COMMENTED** [review]
Very nice!

File: src/server/database/clips.ts [hunk 1]
```diff
@@ -208,17 +217,93 @@ export default class Clips {
...
+     * Counts the amounts of clips with the given sentence id
+     * and updates the clips_count in the sentences table accordingly
+     * @param sentenceId the sentence to update the count for
+     */
     increaseClipCountOnSentence = async (sentenceId: string): Promise<void> => {
         await this.sql.query(
             `
             UPDATE
                 sentences
             SET
```
Comment (staffanru) [inline src/server/database/clips.ts:301]: this should be more accurate than the just count +1
we hade some issues that during a while this field was not updated

Comment (SmariF89) [reply src/server/database/clips.ts:301]: I take it it's working correctly now?

Comment (staffanru) [reply src/server/database/clips.ts:301]: Yes should be, I trust this function more. But the original issue was that this field was not updated at all. I'm still a bit doubtful that it is the best solution to manually update a count field in the db. We should probably rely on a view or something that produces the numbers for us.

But this works, so it is for later if we want to make things prettier :)

**Pull request merged**